src/tests/nativeaot/SmokeTests/HardwareIntrinsics/X64Avx512.csproj

```diff
@@ -18,11 +18,9 @@
   <PropertyGroup>
     <CLRTestBashPreCommands><![CDATA[
 $(CLRTestBashPreCommands)
-    if [[ "$OSTYPE" == "linux"* ]]; then
-        if ! grep -q '^flags.*avx512' /proc/cpuinfo 2>/dev/null; then
```

Comment:
Similar `grep` commands are used in number of other .csproj files. Do all of them need the same fix?